pkg/network/setup/netstat_test.go

```diff
@@ -244,37 +244,6 @@ var _ = Describe("netstat", func() {
 		}), "the pod IP/s should be reported in the status")
 	})
 
-	It("should add a new interface based on the domain spec", func() {
```

Comment:
I don't understand the commit message. Why such option is not supported? Why if the is no ga at all?

Reply:
The VMI spec is not updated with any interfaces, so having only the domain defined but not the vmi spec is odd.

Later on in the PR, I re-introduces this case in a more direct manner. see [here](https://github.com/kubevirt/kubevirt/pull/6951/commits/b2834e17c0bce7a6b1fca6cfc1768a2fa0a22c76#diff-1c532caac99cc080ba1fb740c75c0ff712ca91509093f55cebdfaefee06ae887R316).

Reply:
I tried to express it better in the commit message.